panel/models/echarts.py

```diff
@@ -54,6 +54,8 @@ def __js_skip__(cls):
 
     data = Nullable(Dict(String, Any))
 
+    options_kwargs = Nullable(Dict(String, Any))
```

Comment:
Personally I would call it `options` instead of `options_kwargs`. Everyone knows what `options` are. But `kwargs` is a *scary* word to newbies.